src/prefect/schedules/filters.py

```diff
@@ -100,6 +100,9 @@ def at_time(t: time) -> Callable[[datetime], bool]:
     """
     Filter that allows events that match a specific time.
 
+    For example, `at_time(datetime.time(4))` would only allow runs during the hour of 4 AM
```

Comment:
I'm not sure this example is strictly correct - I think `at_time(datetime.time(4))` would only allow runs at exactly 4am, but not 4:01am.

Reply:
Ah you're correct - updated